### DISCUSSION TITLE
（求助）纳塔音乐被限流了

### DISCUSSION THREAD

**Discussion by Genshin-Neuvillette** [share121/inter-knot#3353] [其他]
在B站，纳塔音乐出现了投币比点赞高的情况。而且点赞率也不行，如果有可能的话，我希望大家能多点赞，多投币，多评论。谢谢了

**Comment by ATEFSTARS** [comment]
难道大家都投币不点赞😨

**Comment by OttoApocalypse01** [comment]
没用的，B铁了心的要压流量，之前还出现过播放量负增长的情况

**Comment by ybbgh** [comment]
笑死，投币能负增长，B站真是蒸蒸日上<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
